drivers/note/Make.defs

```diff
@@ -36,5 +36,10 @@ ifeq ($(CONFIG_DRIVER_NOTECTL),y)
   CSRCS += notectl_driver.c
 endif
 
+ifeq ($(CONFIG_DRIVER_NOTESCHED),y)
+  CSRCS += notesched_driver.c
+  CFLAGS += ${INCDIR_PREFIX}${TOPDIR}/sched
```

Comment:
let's remove ifeq/endif from `CFLAGS += ${INCDIR_PREFIX}${TOPDIR}/sched` to avoid the dup at line 24

Reply:
done